api/types/constants.go

```diff
@@ -138,6 +138,16 @@ const (
 	// SubKindOpenSSHNode is a registered OpenSSH (agentless) node.
 	SubKindOpenSSHNode = "openssh"
 
+	// SubKindOpenSSHEphemeralKeyNode is a registered OpenSSH (agentless) node that doesn't require trust in Teleport CA.
+	// For each session an SSH Key is created and uploaded to the target host using a side-channel.
+	//
+	// For Amazon EC2 Instances, it uploads the key using:
+	// https://docs.aws.amazon.com/ec2-instance-connect/latest/APIReference/API_SendSSHPublicKey.html
+	// This Key is valid for 60 seconds.
+	//
+	// It uses the private key to SSH into the host.
```

Comment:
Whose private key?

Reply:
We create an SSH Key and upload it to the EC2 instance.
We then use the private key (that was created above) to access the EC2 instance over ssh.

I guess this was not very clear :thinking: 
Should I rephrase?

Reply:
I think if it said "It uses the private key created above to SSH into the host" I'd already get it.